modules/nw-ovn-kubernetes-checking-live-migration-metrics.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="checking-live-migration-metrics"]
+= Checking limited live migration metrics 
+
+Metrics are available to monitor the progress of the limited live migration. Metrics can be viewed on the {product-title} web console, or by using the `oc` CLI.
```

Comment:
There are a lot of instances throughout the PR of "the `oc` CLI" that could instead be:
```suggestion
Metrics are available to monitor the progress of the limited live migration. Metrics can be viewed on the {product-title} web console, or by using the {oc-first}.
```